doc/applications.md

```diff
@@ -0,0 +1,159 @@
+# Creating applications
+
+RxJS is a great tool to keep your code less error prone. It does that by using pure and stateless functions. But applications are stateful, so how do we bridge the stateless world of RxJS with the stateful world of our applications?
+
+Let us create a simple state store of the value `0`. On each click we want to increase that count in our state store.
+```js
+const button = document.querySelector('button');
```

Comment:
Not all JavaScript programmers are comfortable with the latest ES2015 features. Can you replace all `const` and `let` with `var`? We've been using `var` consistently in the documentation.